src/Wallabag/CoreBundle/Resources/config/services.yml

```diff
@@ -40,11 +40,40 @@ services:
         class: Graby\Graby
         arguments:
             - { error_message: false }
+            - "@wallabag_core.guzzle.http_client"
+            - "@wallabag_core.graby.config_builder"
         calls:
             - [ setLogger, [ "@logger" ] ]
         tags:
             - { name: monolog.logger, channel: graby }
 
+    wallabag_core.graby.config_builder:
+        class: Graby\SiteConfig\ConfigBuilder
```

Comment:
Using an external `ConfigBuilder` doesn't allow configuration overload thru graby.
So you should inject a custom logger here (using `setLogger`) like for graby service